tiatoolbox/wsicore/wsireader.py

```diff
@@ -1622,53 +1631,36 @@ def read_bounds(
         interpolation="cubic",
         pad_mode="constant",
         pad_constant_values=0,
-        location_is_at_requested=False,
+        coords_space='baseline',
         **kwargs,
     ):
 
-        # Note: In this case, `resolution` will be mapped to corresponding
-        # mpp value (We use it as basic because it has already been aligned) to
-        # allow correct scaling wrt to units being of ['baseline', 'level'].
-        # The output of `read_bounds` is therefore should match with output of
-        # the source reader using the same requested resolution.
```

Comment:
I'm not really sure what you mean by this. Why is resolutions being mapped to an mpp?

Reply:
This has been removed. This was when the method used in conjunction with `attach_to_reader`.